shared-types/account.d.ts

```diff
@@ -1,7 +1,7 @@
 interface IAccount {
   _id?: any;
   name: string;
-  type: "cash" | "savings" | "investment" | "credit";
+  type: "cash" | "savings" | "investment" | "credit" | "loan";
```

Comment:
`loan` should be ignored from total balance calculation... but not a deal breaker for this one